pages/partials/questionScorePanel.ejs

```diff
@@ -50,6 +50,12 @@
     </tbody>
   </table>
 
+  <% if (course.help_link_enabled && assessment.help_link) {%>
+    <div class="card-footer">
+      <%- include('./helpLinkPanel') %>
```

Comment:
Taking a second look at this: the `helpLinkPanel` partial is only ever used in one place, so there's not much value in splitting it into a separate file. Consider inlining that HTML here.

Reply:
Oh yes, it did end up only getting used once, I'll inline it.

Reply:
Done! [2f21433](https://github.com/PrairieLearn/PrairieLearn/pull/6141/commits/2f21433b7a7b9f4e87e3edcdb836c08fbb5a4999)